types/README.md

```diff
@@ -0,0 +1,11 @@
+# Language Server Runtimes Types
+
+Language Server Runtimes Types is a package containing types used by the [Language Server Runtimes](../runtimes/) package.
+
+
+This package is independent of [VSCode Protocol](https://github.com/microsoft/vscode-languageserver-node/tree/main/protocol) type definitions.
```

Comment:
nit: would be better to make this documentation a bit more instructive, saying that types should not depend on implementation details from `protocol`.

Reply:
(fixed inline)